initial_pos_list: [[-0.22, 0.30, 0.33], [-0.22, 0, 0.33], [-0.22, -0.30, 0.33], [-0.22, 0.30, 0.08], [-0.22, 0, 0.08], [-0.22, -0.30, 0.08]]
initial_rot_list: [[0,0,0], [0,0,0], [0,0,0], [0,0,0], [0,0,0], [0,0,0]]
initial_quat_list: [[0,0,0,0], [0,0,0,0], [0,0,0,0], [0,0,0,0], [0,0,0,0], [0,0,0,0]]
dimensions: [[0.37, 0.23, 0.20],[0.37, 0.27, 0.20],[0.37, 0.23, 0.20], [0.37, 0.23, 0.18],[0.37, 0.27, 0.18],[0.37, 0.23, 0.18]]
frame_id_list: [kiva_pod_base, kiva_pod_base, kiva_pod_base, kiva_pod_base, kiva_pod_base, kiva_pod_base]
prefixes: [bin_a, bin_b, bin_c, bin_d, bin_e, bin_f]
camera_directions: [x, x, x, x, x, x]
